### PR TITLE
fix(github): Fix accept headers for Github HTTP wrapper

### PR DESCRIPTION
**Pull request by zharinov**
Ref #6686

### CODE REVIEW

File: lib/util/http/github.ts [hunk 1]
```diff
@@ -140,6 +140,22 @@ interface GraphqlOptions {
   fromEnd?: boolean;
 }
 
+function constructAcceptString(input?: any): string {
+  const defaultAccept = 'application/vnd.github.v3+json';
+  const appModeAccept = 'application/vnd.github.machine-man-preview+json';
+  const acceptStrings = typeof input === 'string' ? input.split(/\s*,\s*/) : [];
```
Comment (viceice) [inline lib/util/http/github.ts:146]: Why do we need the type check? Shouldn't it always be a string? Or `null` or `undefined`?

Comment (zharinov) [reply lib/util/http/github.ts:146]: Don't like this too, but generic http header can be also `string[]`or `number`

**Comment by rarkins** [comment]
It's not for creating or updating PRs

File: lib/platform/github/index.ts [hunk 2]
```diff
@@ -746,7 +747,12 @@ export async function getPr(prNo: number): Promise<Pr | null> {
   );
   const pr = (
     await githubApi.getJson<any>(
-      `repos/${config.parentRepo || config.repository}/pulls/${prNo}`
+      `repos/${config.parentRepo || config.repository}/pulls/${prNo}`,
+      {
+        headers: {
+          accept: 'application/vnd.github.merge-info-preview+json',
```
Comment (rarkins) [inline lib/platform/github/index.ts:753]: I think it is applicable to GraphQL only, so is not required here

**Review by rarkins — CHANGES_REQUESTED** [review]

File: lib/platform/github/index.ts (hunk 2 above)
Comment (zharinov) [reply lib/platform/github/index.ts]: Yes, removed it

**Pull request merged**